common/lib/dependabot/errors.rb

```diff
@@ -12,13 +16,30 @@ def initialize(msg = nil)
     private
 
     def sanitize_message(message)
-      return unless message
+      return unless message && message.is_a?(String)
```

Comment:
What could the message be, if not a string?

Reply:
We use some custom errors that raise with an array for some error context but rolled back the use of DependabotError in these cases but wondering if it's a good idea to keep the check in place in case we re-use in future. Seems sad to bomb out because the error serialiser raised an exception.